vllm/engine/arg_utils.py

```diff
@@ -202,6 +203,13 @@ def add_cli_args(
                             help='maximum context length covered by CUDA '
                             'graphs. When a sequence has context length '
                             'larger than this, we fall back to eager mode.')
+        parser.add_argument(
+            "--device",
+            type=str,
+            default="cuda",
+            choices=["cuda"],
+            help=
+            'device type for vLLM execution, supporting CUDA only currently.')
```

Comment:
I don't think we need this argument. I believe the device should be "detected" instead of being declared by the users.

Reply:
I think this may be necessary in the future. Image that if vllm have both cuda and cpu device supported, and user maybe want to run on cpu even if cuda path works on his environment.

Reply:
Makes sense. Thanks for the explanation!